cmd/youtubedr/download.go

```diff
@@ -7,17 +7,57 @@ import (
 	"os/exec"
 	"strings"
 
+	"github.com/kkdai/youtube/v2"
 	"github.com/spf13/cobra"
 )
 
+var (
+	errNotID = fmt.Errorf("cannot detect ID in given input")
+)
+
 // downloadCmd represents the download command
 var downloadCmd = &cobra.Command{
 	Use:     "download",
-	Short:   "Downloads a video from youtube",
+	Short:   "Downloads a video or a playlist from youtube",
 	Example: `youtubedr -o "Campaign Diary".mp4 https://www.youtube.com/watch\?v\=XbNghLqsVwU`,
 	Args:    cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
-		exitOnError(download(args[0]))
+		if playlistID, err := youtube.ExtractPlaylistID(args[0]); err != nil {
+			if videoID, err := youtube.ExtractVideoID(args[0]); err != nil {
+				exitOnError(errNotID)
```

Comment:
Please try to keep the code more on the left. You don't need the else if you write instead:
```go
videoID, err := youtube.ExtractVideoID(args[0])
if err != nil {
	exitOnError(errNotID)
}
```